aggregator/src/aggregator/aggregation_job_creator.rs

```diff
@@ -1412,27 +1457,15 @@ mod tests {
             2,
             100,
             5000,
+            janus_messages::Duration::from_seconds(3600),
         ));
         Arc::clone(&job_creator)
             .create_aggregation_jobs_for_task(Arc::clone(&task))
             .await
             .unwrap();
 
         // Verify -- we haven't received enough reports yet, so we don't create anything.
-        let want_ra_states: Arc<HashMap<_, _>> = Arc::new(
```

Comment:
`read_and_verify_aggregate_info_for_task()` does not panic if the HashMap of report aggregations has extra entries, only if the database has extra rows. At this point in the test, there should be no such rows, so I cleaned this up to be a bit more strict.